src/org/etools/j1939_84/bus/j1939/J1939.java

```diff
@@ -129,6 +129,12 @@
         return e.right.stream().anyMatch(p -> p.getResponse() == Response.BUSY);
     }
 
+    /** Used for development to detect DMs that are manually parsed. */
+    static public boolean isManual(int pgn) {
```

Comment:
Since this is only use in the `main()` method, I'd recommending move this to that class.  It doesn't really belong here.

Reply:
It belongs in J1939, because J1939 is the only place enumerating the manually defined DMs.  I moved main to a test.